docs/network protocol.md

```diff
@@ -755,6 +755,43 @@ GameData is a **dict** but contains these keys and values. It's broken out into
 | location_name_to_id | dict[str, int] | Mapping of all location names to their respective ID.                                                                         |
 | checksum            | str            | A checksum hash of this game's data.                                                                                          |
 
+
+### Data Package Shared Cache
+The shared cache location is `user_cache_dir/datapackage`. `user_cache_dir` varies by platform. The
```

Comment:
I think this needs an extra comment that the client should keep its own cache if the shared cache is inaccessible, such as in webbrowser context.